benchmark/storage/tuple_access_strategy_benchmark.cpp

```diff
@@ -43,6 +43,7 @@ static void BM_SimpleInsert(benchmark::State &state) {
   // We'll say it 2x RawBlock because we zero it, and then populate it. This is likely an underestimation
   size_t bytes_per_repeat = 2 * sizeof(storage::RawBlock);
   state.SetBytesProcessed(state.iterations() * bytes_per_repeat);
+  state.SetItemsProcessed(state.iterations() * layout.num_slots_);
```

Comment:
Maybe document why we're counting this

Reply:
Maybe in a separate PR?

Reply:
I get it now, should be fine. (that you're only telling gbench about another metric, not doing anything new)